build.gradle

```diff
@@ -52,8 +52,8 @@ ext {
     shedlockVersion = '4.43.0'
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = JavaVersion.VERSION_17
+targetCompatibility = JavaVersion.VERSION_17
```

Comment:
@snopoke wanted to highlight that we are going with setting source and target both to Java 17 since the code changes are very minimum in these PRs.

Reply:
It does as it requires the env to have Java 17 installed, but nothing major beyond that